rspec-mode.el

```diff
@@ -820,7 +826,8 @@ or a cons (FILE . LINE), to run one example."
         (chruby-use-corresponding))
 
     (let ((default-directory (or (rspec-project-root) default-directory))
-          (compilation-buffer-name-function 'rspec-compilation-buffer-name))
+          (compilation-buffer-name-function (and rspec-allow-multiple-compilation-buffers
+                                                 'rspec-compilation-buffer-name)))
```

Comment:
Should we also remove the advice?

Reply:
@dgutov I'm not sure what you mean - could you provide an example?

Reply:
@imtayadeway Sorry, please disregard this. I was looking at the diff of an old PR.